java-runtimes-common/test-spring-application/src/main/java/com/google/cloud/runtimes/Application.java

```diff
@@ -2,10 +2,18 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.support.SpringBootServletInitializer;
 
 @SpringBootApplication
-public class Application {
+public class Application extends SpringBootServletInitializer {
```

Comment:
`SpringBootServletInitializer` is required for the WAR, I understand that.  Just to make sure, did you test that the executable JAR is still working with the embedded server?

Reply:
I did run integration tests with the new test app, and it appears to work the same as before in JAR form. From the [spring boot javadocs](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/web/support/SpringBootServletInitializer.html), the language they use seems to imply that it doesn't break anything to add this, and my testing seems to indicate the same.